spec/models/instrument_for_cart_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe InstrumentForCart do
+
+  let(:facility) { FactoryBot.create(:facility) }
+  let(:facility_account) { facility.facility_accounts.create(FactoryBot.attributes_for(:facility_account)) }
+
+  let(:instrument) do
+    FactoryBot.create(:instrument, facility: facility, facility_account: facility_account, no_relay: true).tap do |instrument|
```

Comment:
I'm pretty sure you don't need the `tap` here. FactoryBot's `create` method should take a block yielding the object being created.